types/tests/programmaticApi.test-d.ts

```diff
@@ -1,23 +1,40 @@
 // New Programmatic APi
 import { expectType } from 'tsd';
-import Nightwatch from '..';
+import Nightwatch, { NightwatchAPI, NightwatchClient, NightwatchProgrammaticAPIClient } from '..';
 
 const client = Nightwatch.createClient({
   browserName: 'firefox',
   headless: true,
 });
+Nightwatch.createClient();
+Nightwatch.createClient({});
 
-expectType<typeof Nightwatch>(client);
+expectType<NightwatchProgrammaticAPIClient>(client);
 
-const runner = Nightwatch.CliRunner();
-expectType<typeof Nightwatch>(runner);
+// test methods/properties exported on Nightwatch
+new Nightwatch.by('css selector', 'hello');
+new Nightwatch.Capabilities();
+expectType<NightwatchAPI>(Nightwatch.browser);
+expectType<NightwatchAPI>(Nightwatch.app);
+Nightwatch.Key.NULL;
```

Comment:
Do we want to expectType here?

Reply:
Nope, just checking if `NULL` exists on `Nightwatch.Key`.